### DISCUSSION TITLE
error while installing package on laravel 8

### DISCUSSION THREAD

**Discussion by shuujin23** [kreait/firebase-php#581] [Q&A]
Hello, 
i got some error while installing on laravel 8. The error look like this
![image](https://user-images.githubusercontent.com/8181423/112091851-dd73b580-8bc8-11eb-9702-3d2006adb61a.png)

and this is my composer.json 
![image](https://user-images.githubusercontent.com/8181423/112091895-f2504900-8bc8-11eb-8594-73edef101085.png)

is there any requirement before installing? 
Thank you

**Comment by jeromegamez** [comment]
tymon/jwt-auth is preventing the installation of lcobucci/jwt (https://github.com/tymondesigns/jwt-auth/releases/tag/1.0.2), unfortunately, this is a conflict that can't be resolved right now, unless you're willing to install an older release of the SDK.

The newest release you can use is 5.11.0, so I think if you first do a `composer require "kreait/firebase-php:<5.12.0"`, it should work.

I've written  `<5.12.0` instead of `5.11.0` because I think it shows more clearly that it's an old version, at least for me, so that I'm constantly reminded to check if there's a new tymon/jwt-auth release 😅.

The actual solution would be if tymon/jwt-auth would be updated (it currently can't be used with PHP 8 as well), there are a couple of requests for that in https://github.com/tymondesigns/jwt-auth/issues already. 🤞

**Reply by irvine48** [reply]
not using jwt-auth, but facing same issue when install laravel-firebase.
**jmespath/jmespath.php failed**

sanctum related?

**Reply by jeromegamez** [reply]
jmespath.php has no dependencies that I can see that could conflict with lcobucci/jwt, could you share some details on what "failed"?

**Reply by irvine48** [reply]
![image](https://user-images.githubusercontent.com/6119065/115430796-433e7600-a237-11eb-81c4-66ac25495f79.png)

i try open another fresh copy of laravel 8.
i am not sure about this, but this keep coming up when install.

**Reply by irvine48** [reply]
found the problem related to jwt 3.3.3

solution as below,
try to lock your firebase-tokens: <1.12.0
try to lock your firebase-php:<5.12.0

it should be installable along laravel passport.
